apps/dashboard/test/config/configuration_singleton_test.rb

```diff
@@ -92,20 +92,22 @@ def no_config_env
 
   test "should load environment variable local overloads" do
     Dir.mktmpdir do |dir|
+      environment = "test"
       ENV['FOO'] = '123'
 
       Configuration
 
-      Pathname.new(dir).join(".env.#{Rails.env}.overload").write <<-EOT
+      Pathname.new(dir).join(".env.#{environment}.overload").write <<-EOT
```

Comment:
I wonder why this stopped working?

Reply:
Oh I see you had to stub `rails_env` - probably should move to `with_modified_env` but we can leave it for another time.